doc/source/user_guide/ug_troubleshooting.rst

```diff
@@ -138,11 +145,11 @@ conflicts in module dependencies.
 
 `print\_points` (**ice\_in**)
     If true, print numerous diagnostic quantities for two grid cells,
-    one near the north pole and one in the Weddell Sea. This utility
+    defined by `lonpnt` and `latpnt` in the namelist file.
```

Comment:
I understand that this will be generalized so that the user alternatively can insert the indices directly, perhaps in a later PR.